content/getting-started/index.md

```diff
@@ -1,12 +1,14 @@
 ---
 sidebar: getting-started
-next: "/getting-started/setting-up/"
+next: "/getting-started/registering/"
 ---
 
 # Introduction
 
-<!--
-TODO Briefly introduce what Codewars is, get the reader excited.
-TODO Summarize what the reader can learn by reading "Getting Started" pages.
-TODO Add a link to creating a new account at the end of the page just in case the reader haven't registered.
--->
+Welcome to Codewars!
+
+Codewars is a platform which helps you learn, train, and improve your coding skills by solving programming tasks of many types and difficulty levels. You can choose how you would like to learn: do you want to face challenges of increasing difficulty? Or maybe you prefer training by repetition and improving your solutions? After solving a task, you can compare your answer with other users and learn from them too, or you can help less experienced users and answer their questions.
+
+Going through this tutorial, you will learn how to create your new account on Codewars, set it up for optimal training experience, and get familiar with basic elements of user interface. Very soon you will be able to face challenges collected in Codewars library and start learning!
+
+If you haven't joined yet, go to [Sign Up](https://www.codewars.com/join) page and continue reading this tutorial to get you ready.
```

Comment:
This sentence should link to the "Registering" doc and the next navigation should go to the account set up. It's awkward to have this sentence and the next document is about registering.